core/src/wsv.rs

```diff
@@ -162,9 +159,7 @@ impl WorldStateView {
     #[inline]
     pub fn new(world: World, kura: Arc<Kura>) -> Self {
         // Added to remain backward compatible with other code primary in tests
```

Comment:
unfortunately no because it's used in integration benches which are compiled into a different crate. `cfg(test)` only works for unit tests

Reply:
One workaround which I don't think we ever considered was separating tests by `debug` status. We're not currently running tests in `release`, and if we did, we shouldn't be using private API anyway.

Reply:
> One workaround which I don't think we ever considered was separating tests by debug status. We're not currently running tests in release, and if we did, we shouldn't be using private API anyway.

this might be a good idea. We could rely on `debug_assertions` to separate tests that test API that is kinda private